### DISCUSSION TITLE
Can't register tree command: Command 'foo' already registered.

### DISCUSSION THREAD

**Discussion by MeHaVr** [Rapptz/discord.py#9691] [Q&A]
### Why does it tell me 'foo' is already there? It was not registered anywhere else.

```
    @commands.Cog.listener()
    async def on_ready(self):
        await tree.sync(guild=discord.Object(id=1180536174633304184))
        print("ping.py says hello")

    @bot.tree.command()
    async def foo(self, interaction: discord.Interaction):
        await interaction.response.send_message('You must be the owner to use this command!')
```

Error:

`discord.app_commands.errors.CommandAlreadyRegistered: Command 'foo' already registered.`

**Comment by Rapptz** [comment]
Don't use `@bot.tree.command()` in a cog, use `@app_commands.command()` instead.

**Reply by MeHaVr** [reply]
Okay, thanks

But there is still a problem, that the command is not found:

`discord.ext.commands.errors.CommandNotFound: Command "/foo" is not found`

I checked that the command is in the `tree` before sync.

**Reply by Rapptz** [reply]
That's an `ext.commands` error which is when you call something with a prefix like e.g. `?foo`. You are most likely doing `[p]/foo` where `[p]` is your prefix.

**Reply by MeHaVr** [reply]
No, I just type` /foo `

The bot was created like so: 
```
import discord
from discord.ext import commands
from discord import app_commands
print("bot setup")
intents = discord.Intents.default()
intents.members = True
intents.message_content = True
bot = commands.Bot(command_prefix='/', intents=intents)
```

**Reply by Rapptz** [reply]
Your prefix shouldn't be /

**Reply by MeHaVr** [reply]
What prefixes are allowed? Ist it possible to use / for bot commands at all?

**Reply by itslychee** [reply]
it's not practical to use / for prefix commands when it's used for slash commands.

**Reply by MeHaVr** [reply]
How can I use slash commands?

**Reply by Link1O** [reply]
Firstly, refrain from syncing at the bot's startup; instead, utilize a prefix command for this action. Regarding the question 'How can I use slash commands?'—it's rather unclear what you're trying to convey.
